cockpit-composer.spec.in

```diff
@@ -11,8 +11,12 @@ BuildArch:      noarch
 BuildRequires:  libappstream-glib
 
 Requires:       cockpit
-Requires:       weldr
-Suggests:       osbuild-composer >= 14
+%if 0%{?fedora} > 32 || 0%{?rhel} >= 8
```

Comment:
I suggest `>= 33`, I find it more consistent.

Reply:
Okay. @gicmo said that 33 currently doesn't exist so I wasn't sure if I should use it.

Reply:
Might have been wrong about it, was not sure since we are not branched yet.

Reply:
Even if we aren't yet, that's fine -- if your intention is that "From Fedora N on, we'll only support osbuild", then it should say `>= N`.

Reply:
FTR, I was talking about `%{fc#}` when I said that I was unsure if it was not defined yet, not about `%{fedora}`, but it of course is defined (`fc33` is defined to `1`) on rawhide because the release is 33 in the [`fedora-release.spec`](https://src.fedoraproject.org/rpms/fedora-release/blob/master/f/fedora-release.spec#_741) for rawhide.

Reply:
Okay thanks for all of the info. This has been updated to >=33